github-api/src/main/java/org/eclipse/egit/github/core/service/IssueService.java

```diff
@@ -47,7 +30,24 @@
 import org.eclipse.egit.github.core.client.PagedRequest;
 import org.eclipse.egit.github.core.util.UrlUtils;
 
-import com.google.gson.reflect.TypeToken;
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.eclipse.egit.github.core.client.IGitHubConstants.CHARSET_UTF8;
+import static org.eclipse.egit.github.core.client.IGitHubConstants.SEGMENT_COMMENTS;
+import static org.eclipse.egit.github.core.client.IGitHubConstants.SEGMENT_EVENTS;
+import static org.eclipse.egit.github.core.client.IGitHubConstants.SEGMENT_ISSUES;
+import static org.eclipse.egit.github.core.client.IGitHubConstants.SEGMENT_LEGACY;
+import static org.eclipse.egit.github.core.client.IGitHubConstants.SEGMENT_REACTIONS;
+import static org.eclipse.egit.github.core.client.IGitHubConstants.SEGMENT_REPOS;
+import static org.eclipse.egit.github.core.client.IGitHubConstants.SEGMENT_SEARCH;
+import static org.eclipse.egit.github.core.client.PagedRequest.PAGE_FIRST;
+import static org.eclipse.egit.github.core.client.PagedRequest.PAGE_SIZE;
```

Comment:
Reordering seems unneeded (yes, I agree it's annoying, I just want to keep the diff to the original bindings as small as possible).

Reply:
Every single time before I commit I view whole diff and check if there is anything that I've missed, yet I still manage to always get these import changes unnoticed.

I probably have a checkbox to auto organize them enabled in a commit dialog... Will check and fix that later 😆

(I wish there was an option to use different styling rules per module in AS)

Reply:
Fixed and disabled the checkbox so I shouldn't make that mistake again.